### DISCUSSION TITLE
Animation Test with lighting and comp

### DISCUSSION THREAD

**Discussion by diegostoliar** [keiwando/toonsquid#486] [Show and tell]
[Animation Test with lighting and comp](https://sites.google.com/view/stoliar-art/watch/anim001)

Here is a little test in Toonsquid with lighting and comp... I'm just testing a style, so the goal was not continuity or model, but just the overall look for now.

Hope you like it!
Diego

**Comment by Cat-Butler** [comment]
Hi Diego, I'm getting a YouTube error that this video is private (unavailable). 
Regards, Butler to Cats. 🤵🏻🚪🐈

**Comment by diegostoliar** [comment]
gah, let me fix that

On Wed, Dec 13, 2023 at 4:08 PM Butler_to_Cats ***@***.***>
wrote:

> Hi Diego, I'm getting a YouTube error that this video is private
> (unavailable).
> Regards, Butler to Cats. 🤵🏻🚪🐈
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/keiwando/toonsquid/discussions/486#discussioncomment-7848221>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACNFHN7QT4KPVG5UIDM32E3YJI7RTAVCNFSM6AAAAABASS2ZFWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQNBYGIZDC>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Cheers,
Diego Stoliar
www.stoliar.art

**Comment by diegostoliar** [comment]
should be viewable now! Thanks for the flag...

On Thu, Dec 14, 2023 at 11:02 AM Diego Stoliar ***@***.***>
wrote:

> gah, let me fix that
>
> On Wed, Dec 13, 2023 at 4:08 PM Butler_to_Cats ***@***.***>
> wrote:
>
>> Hi Diego, I'm getting a YouTube error that this video is private
>> (unavailable).
>> Regards, Butler to Cats. 🤵🏻🚪🐈
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/keiwando/toonsquid/discussions/486#discussioncomment-7848221>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ACNFHN7QT4KPVG5UIDM32E3YJI7RTAVCNFSM6AAAAABASS2ZFWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQNBYGIZDC>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
>
> --
> Cheers,
> Diego Stoliar
> www.stoliar.art
>


-- 
Cheers,
Diego Stoliar
www.stoliar.art

**Reply by cvanalphen** [reply]
I still get a 404 error after clicking on the link ….

**Reply by diegostoliar** [reply]
Sorry, fixed once more

**Comment by joelmayer87** [comment]
Love it @diegostoliar :) Really silly question but how did you color the character when using a textured line like this? Did you do it manually by just brushing it in or is there some neat shortcut that lets you sensibly use the fill option? Love the style!

**Reply by diegostoliar** [reply]
First I use the auto fill option on the pencil, than I do a color at a time, first the skin color for the whole shape of the character, than I lock the transparency when doing the next color which speeds up the process. If I think a color is hard I do it in another layer and clip mask it to the skin color layer, than its easier to erase if needed. At the end I merge all color layers to make it lighter and easier to comp them.

**Reply by diegostoliar** [reply]
Here is another answer for you Joel,

You can also try doing it with vector shapes. Very similar workflow: first the main color, and then other color on top shapes using clipping mask.
The advantage of doing vector shapes is that after you set up one shape you can just adjust its contour frame by frame without having to redraw the whole contour, you can tween the shapes, put the in hold, change the shape color for all frames after the fact, duplicate and move shapes etc... so quite a bit of advantages.

You can also have multiple shapes in a single layer, which also saves quite a bit of space.

I'm trying this workflow more and more and enjoying it quite a bit.

**Comment by Cat-Butler** [comment]
I'm reminded of Disney's animated Tarzan.
Nicely done.

**Reply by diegostoliar** [reply]
S2
